tests/gnrc_ipv6_fwd_w_sub/mockup_netif.c

```diff
@@ -69,10 +69,11 @@ void _tests_init(void)
         );
     _mock_netif = &_netif;
     expect(res == 0);
+
     gnrc_ipv6_nib_init();
-    gnrc_netif_acquire(_mock_netif);
     gnrc_ipv6_nib_init_iface(_mock_netif);
-    gnrc_netif_release(_mock_netif);
+    gnrc_ipv6_nib_iface_up(_mock_netif);
```

Comment:
`gnrc_ipv6_nib_init()`/`gnrc_ipv6_nib_iface_up()` will both call `gnrc_netif_acquire()` - no need to do it here.